tasks/main.yaml

```diff
@@ -8,6 +8,7 @@
 - name: Check for Tuned legacy
   shell: tuned --version &> /dev/null || echo -n "legacy"
   register: tversion
+  check_mode: no
```

Comment:
what about using `changed_when: false` instead, or in addition to `check_mode: no`?